src/backend/cdb/motion/ic_udpifc.c

```diff
@@ -1185,6 +1186,7 @@ setupUDPListeningSocket(int *listenerSocketFd, uint16 *listenerPort, int *txFami
 	uint32					socketSendBufferSize;
 	uint32					socketRecvBufferSize;
 
+	snprintf(service, 32, "%d", 0);
```

Comment:
`setupTCPListeningSocket()` contains a [comment](https://github.com/arenadata/gpdb/blob/2f9ad0b2d5ee6f2a9ac8361d48bc4237d1db9b69/src/backend/cdb/motion/ic_tcp.c#L151) about similar line. Maybe add here the same comment as well?